src/commands/game_commands/play.ts

```diff
@@ -109,62 +107,92 @@ export default class PlayCommand implements BaseCommand {
-                    const startTitle = `\`${process.env.BOT_PREFIX}join\` the game and start it with \`${process.env.BOT_PREFIX}begin\`!`;
-                    const gameInstructions = `Type \`${process.env.BOT_PREFIX}join\` to play in the upcoming elimination game. Once all have joined, ${bold(gameOwner.tag)} must send \`${process.env.BOT_PREFIX}begin\` to start the game. Everyone begins with \`${lives}\` lives.`;
-                    gameSession = new GameSession(textChannel.id, voiceChannel.id, textChannel.guild.id, gameOwner, GameType.ELIMINATION, lives);
-                    gameSession.addEliminationParticipant(gameOwner);
-                    await sendInfoMessage(messageContext, { title: startTitle, description: gameInstructions, thumbnailUrl: KmqImages.HAPPY });
-                } else if (isTeamsMode) {
-                    // (1) TEAMS game creation
-                    const startTitle = `\`${process.env.BOT_PREFIX}join\` a team!`;
-                    const gameInstructions = `Team leaders, type \`${process.env.BOT_PREFIX}join [team name]\` to form a new team. Remember, switching teams mid-game will forfeit all your current score and EXP.`;
-                    await sendInfoMessage(messageContext, { title: startTitle, description: gameInstructions, thumbnailUrl: KmqImages.HAPPY });
-                    gameSession = new GameSession(textChannel.id, voiceChannel.id, textChannel.guild.id, gameOwner, GameType.TEAMS);
+        const isEliminationMode = parsedMessage.components.length >= 1 && parsedMessage.components[0].toLowerCase() === "elimination";
+        const isTeamsMode = parsedMessage.components.length >= 1 && parsedMessage.components[0].toLowerCase() === "teams";
+        const isMcMode = parsedMessage.components.length >= 1 && parsedMessage.components[0].toLowerCase() === "mc";
+
+        if (gameSessions[message.guildID] && !gameSessions[message.guildID].sessionInitialized && (isEliminationMode || isTeamsMode)) {
+            // User sent ,play elimination or ,play teams twice, reset the GameSession
+            deleteGameSession(message.guildID);
+        }
+
+        const messageContext = MessageContext.fromMessage(message);
+        const prefix = process.env.BOT_PREFIX;
+
+        if (!gameSessions[message.guildID] || !gameSessions[message.guildID].sessionInitialized) {
+            // (1) No game session exists yet (create MC, ELIMINATION, TEAMS, or CLASSIC game), or
+            // (2) User attempting to ,play after a ,play elimination/teams that didn't start, start CLASSIC game
+            const textChannel = channel;
+            const gameOwner = KmqMember.fromUser(message.author);
+            let gameSession: GameSession;
+
+            if (isMcMode) {
+                // (1) MC game creation
+                let difficulty: GameType.MC_EASY | GameType.MC_MEDIUM | GameType.MC_HARD;
+                if (parsedMessage.components.length === 1 || parsedMessage.components[1].toLowerCase() === "easy") {
```

Comment:
Pull `parsedMessage.components[1].toLowerCase()` into a constant. Why is `parsedMessage.components.length === 1` check only done once?

Reply:
`components[0]` is `mc`, and if the user does not pass a difficulty, it defaults to easy.